src/Entity/ApiDoc.php

```diff
@@ -73,63 +78,64 @@
 class ApiDoc extends ContentEntityBase implements ApiDocInterface {
 
   use EntityChangedTrait;
+  use StringTranslationTrait;
```

Comment:
As I can see you are not calling `$this->t()` here this import is unnecessary.
(Because [it silently falls back](https://github.com/drupal/core/blob/8.7.x/lib/Drupal/Core/StringTranslation/StringTranslationTrait.php#L103-L105) to the global static service, it is also considered as bad practice using it in a context where DI is not available.)

Reply:
Good catch. I'll remove the trait.